front/app/containers/Authentication/SignUpIn/SignUpInModal/SignUp/VerificationSignUpStep/VerificationSteps.tsx

```diff
@@ -85,7 +86,8 @@ const VerificationSteps = memo<Props>(
 
     const goToSuccessStep = useCallback(() => {
       if (!isNilOrError(authUser)) {
-        streams.reset().then(() => {
+        streams.reset().then(async () => {
+          await resetQueryCache();
```

Comment:
We could resolve these 2 in parallel, but I don't think it will make much difference.

Reply:
We could, indeed but didn't want to complicate things with promise.all() in this case, especially because we are hopefully deleting the steams reset relatively soon.